.github/workflows/docker.yml

```
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - main
+    tags:
+      - "*"
```

If this does get triggered per tag (i.e. once per package per version release), we could restrict this to just one of the packages we expect to always exist (e.g. `common`).
```suggestion
      - "@latticexyz/common@*"
```